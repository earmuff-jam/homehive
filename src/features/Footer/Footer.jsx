import { Stack, Typography } from "@mui/material";

function Footer() {
  return (
    <Stack sx={{ textAlign: "center" }}>
      <Typography> Earmuffjam LLC. All rights reserved. @2024</Typography>
    </Stack>
  );
}

export default Footer;
