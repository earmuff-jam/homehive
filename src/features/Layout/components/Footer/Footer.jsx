import React from "react";

import { Stack, Typography } from "@mui/material";

function Footer() {
  return (
    <Stack
      sx={{ textAlign: "center", mt: 1, color: "#bbb" }}
      className="no-print"
    >
      <Typography> Earmuffjam LLC. All rights reserved. @2024</Typography>
    </Stack>
  );
}

export default Footer;
