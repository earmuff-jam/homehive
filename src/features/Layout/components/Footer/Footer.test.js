import React from "react";

import Footer from "./Footer";
import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";

describe("Footer component", () => {
  it("renders correctly and matches snapshot", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
  it("renders footer text", () => {
    render(<Footer />);
    expect(
      screen.getByText(/Earmuffjam LLC. All rights reserved. @2024/i),
    ).toBeInTheDocument();
  });
});
