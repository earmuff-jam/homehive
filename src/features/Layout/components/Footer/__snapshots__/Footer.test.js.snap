// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiStack-root no-print css-anfiki-MuiStack-root"
  >
    <p
      class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
    >
       Earmuffjam LLC. All rights reserved. @2024
    </p>
  </div>
</DocumentFragment>
`;
